ios/FluentUI.Demo/FluentUI.Demo/Demos/SegmentedControlDemoController.swift

```diff
@@ -49,15 +56,18 @@ class SegmentedControlDemoController: DemoController {
 
         addTitle(text: "On Brand Pill")
```

Comment:
can we give more details in this section label "equal width " just to be extra clear with contrast?